.github/workflows/performance.yml

```diff
@@ -73,12 +82,14 @@ jobs:
     permissions:
       contents: read
     if: ${{ github.repository == 'WordPress/wordpress-develop' || github.event_name == 'pull_request' }}
-
+    
     steps:
-      - name: Configure environment variables
-        run: |
-          echo "PHP_FPM_UID=$(id -u)" >> $GITHUB_ENV
-          echo "PHP_FPM_GID=$(id -g)" >> $GITHUB_ENV
+      - name: Get PHP-FPM UID and GID
+        id: get-php-fpm-uid
+        run: echo "::set-output name=php-fpm-uid::$(id -u)"
+      - name: Get PHP-FPM GID
+        id: get-php-fpm-gid
+        run: echo "::set-output name=php-fpm-gid::$(id -g)"
```

Comment:
I don't think these should change, particularly since the `::set-output` command [has been deprecated for GitHub actions](https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/).